examples/tacopie/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2016, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-tacopie)
+
+# download tacopie
+hunter_add_package(tacopie)
+
+# now tacopie can be used
+find_package(tacopie CONFIG REQUIRED)
```

Comment:
Sorry for the delay.
I added the config according to the docs.
The example should work just fine now.